bin/sdk/gutenberg.js

```diff
@@ -0,0 +1,56 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+const chalk = require( 'chalk' );
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+
+const __rootDir = path.resolve( __dirname, '../../' );
+const baseConfig = require( path.join( __rootDir, 'webpack.config.js' ) );
+
+exports.compile = args => {
+
+	const options = {
```

Comment:
It's unclear now that these are required:
- `args.mode`: CLI script defaults this to `production`. Defaulting could be moved to this file but it's nice to have CLI indicate default with `--help` 
- `editorScript`: CLI script requires this